charts/atlas-cluster/README.md

```diff
@@ -97,3 +97,27 @@ containers:
            name: my-project-atlas-cluster-admin-user
            key: connectionString.standardSrv
 ```
+
+## Notes:
```

Comment:
Please remove the colon at the end of the title; maybe change the title to "Upgrade Notes"